workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-intro-bottom-nav/ddm-template.ftl

```diff
@@ -0,0 +1,39 @@
+<#assign
+	groupFriendlyURL = themeDisplay.getScopeGroup().getFriendlyURL()
+	groupPathFriendlyURLPublic = themeDisplay.getPathFriendlyURLPublic() + groupFriendlyURL
+	navigationJSONObject = jsonFactoryUtil.createJSONObject(navigation.getData())
+	childrenJSONArray = navigationJSONObject.getJSONArray("children")
+	childrenArrayLength = childrenJSONArray.length()
+/>
+
+	<div>
+		<#if childrenArrayLength gt 0>
+			<ul class="m-0 p-2">
+				<#list 0..childrenArrayLength-1 as i>
+					<#assign child = childrenJSONArray.getJSONObject(i) />
+					<#if i = 1>
+						<li class="course-bottom-nav-item">
+							<a
+								class="d-flex justify-content-between" ${(navigationJSONObject.getJSONObject("self").url == child.url)?then("selected", "")}'
```

Comment:
@luisacmn there appears to be an extra ' at the end of this line.  My editor is complaining about it so can you take a look?

Reply:
Hey @sammonsjl . Sure, thanks.